common/src/main/java/com/github/twitch4j/common/util/TwitchUtils.java

```diff
@@ -45,6 +45,11 @@
             if (badges.containsKey("staff")) {
                 permissionSet.add(CommandPermission.TWITCHSTAFF);
             }
+            // Founder
+            if(badges.containsKey("founder")) {
+                permissionSet.add(CommandPermission.FOUNDER);
+                permissionSet.add(CommandPermission.SUBSCRIBER);
```

Comment:
I read the docs about founders and found this part:
```
Streamers that took extended breaks from streaming may have lost most of their founding
subs and may see little to none of their founders in the chat today.
Those former subscribers will receive emails that they’ve received the
Founders Badge, but there’s no guarantee that they will re-sub.

Regardless, congratulations to the new Founders. You're first forever.
```
= Having the founder badge doesn't mean someone is still subbed, just that they were first - so adding sub permissions is not accurate.

We should still detect founders that are subbed based on the irc tag (not badge) - the code a bit lower:
```
// Subscriber
if (tags.containsKey("subscriber") && tags.get("subscriber").equals("1")) {
  permissionSet.add(CommandPermission.SUBSCRIBER);
}
```

Reply:
Nevermind, you do need a active sub to equip the badge so this is fine.